README.md

```diff
@@ -1,15 +1,54 @@
-# [![Build Status](https://travis-ci.com/SUSE/catapult.svg?branch=master)](https://travis-ci.com/SUSE/catapult) catapult: [SCF](https://github.com/SUSE/scf) on top of [Kind](https://github.com/kubernetes-sigs/kind) (and others..)
+# [![Build  Status](https://travis-ci.com/SUSE/catapult.svg?branch=master)](https://travis-ci.com/SUSE/catapult) CI implementation for [KubeCF](https://github.com/SUSE/kubecf), [SCF](https://github.com/SUSE/scf) & [Stratos](https://github.com/cloudfoundry/stratos)
```

Comment:
I would leave at this point just "Catapult" in the headers no? and we could add `CI implementation for [KubeCF](https://github.com/SUSE/kubecf), [SCF](https://github.com/SUSE/scf) & [Stratos](https://github.com/cloudfoundry/stratos)` as a short info right after , wdyt?

Reply:
true! done!